app/controllers/staff/actuals_controller.rb

```diff
@@ -22,45 +24,45 @@ def new
   def create
     @activity = activity
     authorize @activity
+    @actual = ActualForm.new(actual_params.merge({parent_activity: @activity}))
 
-    result = CreateActual.new(activity: @activity)
-      .call(attributes: actual_params)
-    @actual = result.object
```

Comment:
It looks like we might have changed the semantics with this deletion. Prior to the deletion, we were sending an activerecord object (unpersisted) to `render :new`, but now we're returning an `ActualForm`. I don't _think_ this has a material effect but we can note it.

Reply:
(i.e. not a blocker but not going to resolve until seen 😄 )

Reply:
Narrator: they could not